content/json-ld/latest/events/update-publication-date.md

```diff
@@ -0,0 +1,131 @@
+---
+---
+
+# Update the publication date
```

Comment:
Are we gonna name this publication date or available from?

Reply:
Changed it to available from for consistency 👍